src/styles/base/_base.scss

```diff
@@ -1,7 +1,60 @@
 @use 'variables';
 
+* {
+  margin: 0;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
 body {
   color: variables.$text-color;
 
   font-family: variables.$font-primary-regular;
+  line-height: 1.5;
```

Comment:
Aslında tüm reset/normalize ile ilgili style'ları `_reset.scss` veya `_normalize.scss` gibi bir dosyada tutsak, `_base.scss` tarafında project-specific base'leri alsak daha güzel olabilir.

Örneğin `body` tarafındaki bu `color` ve `font-family` project specific gibi düşünsek, diğerlerinin tamamı reset ise bunu ayırıp devamında `main.scss` en tepesinde `@use` yapabiliriz.

Base de proje devamında büyüyebilir, h tag'lerinin font-size'ları renkleri, `a` `text-decoration: none` vb buraya bir çok base gelebilir.